tools/mkpkg/update_binary-addons

```diff
@@ -152,27 +197,21 @@ for addontxt in $KODI_DIR/cmake/addons/bootstrap/repositories/*-addons.txt ; do
 
     REPO=$(cat $addon/$ADDON.txt | awk '{print $2}')
     GIT_HASH=$(cat $addon/$ADDON.txt | awk '{print $3}')
-    PKG_NAME="$ADDON"
 
     # Verify the Kodi repo matches our package repo
     # If different, ignore the addon and process it later as an "unofficial" addon
     validate_pkg_url "$ADDON" "$REPO" || continue
 
-    PROCESSED="${PROCESSED}${PKG_NAME}\n"
+    PROCESSED="${PROCESSED}${ADDON}\n"
 
     if ! grep -q all $addon/platforms.txt && ! grep -q linux $addon/platforms.txt && ! grep -q ! $addon/platforms.txt; then
       continue
     fi
 
     if [ -f ${ROOT}/packages/mediacenter/kodi-binary-addons/$ADDON/package.mk ] ; then
-      git_clone $REPO ${KODI_BRANCH} $PKG_NAME.git $GIT_HASH
-
-      # update package.mk
-      RESOLVED_HASH=$(resolve_hash $PKG_NAME.git $GIT_HASH)
-      echo "Resolving hash for ${PKG_NAME}: ${GIT_HASH} => ${RESOLVED_HASH}"
-      update_pkg ${ROOT}/packages/mediacenter/kodi-binary-addons/$ADDON/package.mk ${PKG_NAME} ${RESOLVED_HASH}
-
-      rm -rf $PKG_NAME.git
+      update_to_latest_tag \
+        ${ROOT}/packages/mediacenter/kodi-binary-addons/${ADDON}/package.mk \
```

Comment:
```suggestion
        "${ROOT}/packages/mediacenter/kodi-binary-addons/${ADDON}/package.mk" \
```

Just in case `${ROOT}` includes spaces...

Reply:
will do

Reply:
oh, there are tons of unquoted more ROOT references, will look at them as well